Release/6.16.0 (#1653)
onfido/onfido-sdk-ui: README.md

  If `useModal` is set to `true`, by default the user can close the SDK by clicking on the close button or on the background overlay. You can disable the user's ability to close the SDK by clicking the background overlay through setting `shouldCloseOnOverlayClick` to `false`.

- **`autoFocusOnInitialScreenTitle {Boolean} optional (default: true)`**

>   Set SDK to auto focus on the initial screen's title. By default it will auto focus on every screen's title. When disabled, auto focus will not be applied for the initial screen's title. SDK will still auto focus to subsequent screens' title as user goes through the steps.

Reviewer: ```suggestion
  Sets the SDK to auto focus on the initial screen's title. By default the SDK will auto focus on every screen's title. When disabled, auto focus will not be applied for the initial screen's title. The SDK will still auto focus to all subsequent screens' titles as the user goes through the steps.
```
